.github/workflows/pr_dependency.yml

```
@@ -0,0 +1,16 @@
+# https://github.com/marketplace/actions/pr-dependency-check
+name: "PR Dependency Check"
+on:
+  pull_request:
+  issue_comment:
+    types: [ created ]
+
+jobs:
+  check_dependencies:
+    runs-on: ubuntu-latest
+    if: (github.event.issue.pull_request != '' && contains(github.event.comment.body, 'merge')) || github.event_name == 'pull_request'
+    name: Check Dependencies
+    steps:
+    - uses: gregsdennis/dependencies-action@main
```

```suggestion
    - uses: gregsdennis/dependencies-action@v1.4.1
```

We should pin action version